asciidoctorj-core/src/main/java/org/asciidoctor/log/internal/JULLogHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.asciidoctor.log.internal;
+
+import org.asciidoctor.log.LogHandler;
+import org.asciidoctor.log.LogRecord;
+import org.asciidoctor.log.Severity;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class JULLogHandler implements LogHandler {
+
+    private final static Logger LOGGER = Logger.getLogger("asciidoctor");
+
+    @Override
+    public void log(LogRecord logRecord) {
+        final java.util.logging.LogRecord julLogRecord =
+            new java.util.logging.LogRecord(
+                mapSeverity(logRecord.getSeverity()),
+                logRecord.getCursor() != null ? logRecord.getCursor().toString() + ": " + logRecord.getMessage() : logRecord.getMessage());
+
+        julLogRecord.setSourceClassName(logRecord.getSourceFileName());
```

Comment:
cursor is not captured here.
I fixed it locally adding ` julLogRecord.setParameters(new Object[] { logRecord.getCursor() });`

Reply:
I added this below.
(Just mentioning the obvious here so that I remember that I already handled this comment)